src/QMCHamiltonians/ACForce.h

```diff
@@ -62,13 +62,15 @@ class ACForce : public OperatorBase
  *  isn't sufficient.  We override it here. **/
   void add2Hamiltonian(ParticleSet& qp, TrialWaveFunction& psi, QMCHamiltonian& targetH) final;
 
+  RealType compute_regularizer_f(const ParticleGradient & G, const RealType epsilon);
```

Comment:
Please document this function. Can this function marked static?

Reply:
For sure.  I'll document and make static.

Reply:
Done